### PR TITLE
Integrations: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Integrations: apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/config.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Configuration
 description: Instructions on how to setup the configuration panel for Home Assistant.
 ha_category:
-  - Front End
+  - Front end
```
Comment (ludeeus) [inline source/_integrations/config.markdown:5]: Should this be split?
<https://www.home-assistant.io/integrations/frontend/>

Comment (c0ffeeca7) [reply source/_integrations/config.markdown:5]: yes. Thank you for the catch, I'll fix it on the other page.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/front-end
Also, I  have not forgotten about your feedback from earlier about the term inconsistency on front end, front-end, UI, etc. Just did not get there yet.

File: source/_integrations/panel_custom.markdown [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Custom Panel
+title: Custom panel
 description: Instructions on how to add customized panels to the frontend of Home Assistant.
```
Comment (ludeeus) [inline source/_integrations/panel_custom.markdown]:
```suggestion
description: Instructions on how to add customized panels to the front end of Home Assistant.
```

File: source/_integrations/panel_iframe.markdown [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-title: iframe Panel
+title: iframe panel
 description: Instructions on how to add iFrames in the frontend of Home Assistant.
```
Comment (ludeeus) [inline source/_integrations/panel_iframe.markdown]:
```suggestion
description: Instructions on how to add iFrames in the front end of Home Assistant.
```

File: source/_integrations/panel_iframe.markdown [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
-title: iframe Panel
+title: iframe panel
```
Comment (ludeeus) [inline source/_integrations/panel_iframe.markdown]:
```suggestion
title: iFrame panel
```
MS uses `iframe`, `iFrame` and `IFRAME` for this, not sure which is the correct one.
but `iFrame` matches the description.

**Review by ludeeus — APPROVED** [review]

**Pull request merged**